core/src/main/java/com/netflix/conductor/metrics/Monitors.java

```diff
@@ -278,4 +278,12 @@ public static void recordWorkerQueueSize(String queueType, int val) {
 	public static void recordDiscardedIndexingCount(String queueType) {
 		getCounter(Monitors.classQualifier, "discarded_index_count", "queueType", queueType).increment();
 	}
+
+	public static void recordAcquireLockUnsuccessful(String lockId) {
+		counter(classQualifier, "acquire_lock_unsuccessful", "lockId", lockId);
```

Comment:
`lockId` is too granular of a metric to be used as a tag. A better metric might be to identify which of the acquire lock API's failed and log the lockId for debugging purposes.